README.md

```diff
@@ -2,13 +2,13 @@
 
 ## About
 
-This repository contains Dockerfiles for self-hosted GitHub Actions runners and an associated Terraform module which can be run in your environment to provision:
+This repository contains Dockerfiles for self-hosted GitHub Actions runners and an associated Terraform module, `github-actions-runner-terraform`, which can be run in your environment to provision:
 
 - ECS Cluster
 - ECS Service
 - ECS Fargate task definition to spin up an instance of this runner _per job_ in your GitHub Actions workflow
 
-This module uses an existing ECR repository in AWS, and so does not provision one.
+This module uses the ECR repository running in the MACBIS Shared DSO Dev account, managed in the `terraform/dev/account` directory, and accessible by the MACBIS organization.
```

Comment:
nit: ECR repositories don't 'run' per se

Reply:
Changed "running" to "create"